core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1432,6 +1432,10 @@ public enum Property {
   COMPACTOR_PREFIX("compactor.", null, PropertyType.PREFIX,
       "Properties in this category affect the behavior of the accumulo compactor server.", "2.1.0"),
   @Experimental
+  COMPACTOR_CHECK_MAX_WAIT("compactor.check.max.wait", "5m", PropertyType.TIMEDURATION,
```

Comment:
The following exists in elasticity 

https://github.com/apache/accumulo/blob/baefeb7a5eadfb8af59dddb827a45767c25d5e02/core/src/main/java/org/apache/accumulo/core/conf/Property.java#L1122

and has the same purpose.  Should make this property have the same name OR rename that property in elasticity eventually.

Reply:
I like the `job` reference better than `check` so I'll refactor to use the existing elasticity property name.